splat.yaml

```diff
@@ -64,163 +64,163 @@ segments:
-    - [0x435B0, c] // 
-    - [0x435D0, asm]  // pkm lab cries code
-    - [0x45720, asm]  // music code
-    - [0x46680, asm]  // speech code
-    - [0x47B60, asm]  // sound code
-    - [0x485C0, asm]  // moves code 
-    - [0x48C60, asm]  // move effects code
-    - [0x490A0, asm]  // audio functions to 0x50C00
-    - [0x49190, bin]
-    - [0x49790, asm]
-    - [0x49BA0, asm]
-    - [0x49DC0, asm]
-    - [0x4A360, asm]
-    - [0x4A3E0, asm]
-    - [0x4B940, asm]
-    - [0x4BA90, asm]
-    - [0x4BDC0, asm]
-    - [0x4F410, asm]
-    - [0x4F7F0, asm]
-    - [0x4F870, asm]
-    - [0x4FDB0, asm]
-    - [0x4FE60, asm]
-    - [0x4FF50, bin]
-    - [0x50000, asm]
-    - [0x50120, asm]
-    - [0x50860, asm]
-    - [0x50990, asm]
-    - [0x50A00, asm]
-    - [0x50C20, asm]
-    - [0x50CC0, asm]
+    - [0x32D10, asm] // magikarp minigame point calculations
+    - [0x33E30, asm] // ---UNKNOWN NO EXECUTING
+    - [0x33FE0, asm] // ---UNKNOWN NO EXECUTING
+    - [0x37370, asm] // ---UNKNOWN NO EXECUTING
+    - [0x373A0, asm] // battle code
+    - [0x3D140, asm] // music code
+    - [0x43520, c] // audio flags
+    - [0x435B0, c] // stack cache cleaner
+    - [0x435D0, asm] // pkm lab cries code
+    - [0x45720, asm] // music code
+    - [0x46680, asm] // speech code
+    - [0x47B60, asm] // sound code
+    - [0x485C0, asm] // moves code 
+    - [0x48C60, asm] // move effects code
+    - [0x490A0, asm] // audio functions to 0x50C00
+    - [0x49190, bin] // pokemon cries code 
+    - [0x49790, asm] // menu sound effects 
+    - [0x49BA0, asm] // music sound engine
+    - [0x49DC0, asm] // audio flags
+    - [0x4A360, asm] // audio flags
+    - [0x4A3E0, asm] // audio flags
+    - [0x4B940, asm] // audio flags
+    - [0x4BA90, asm] // audio flags
+    - [0x4BDC0, asm] // clefairy minigame music sequencer flag 
+    - [0x4F410, asm] // pkm cries flags
+    - [0x4F7F0, asm] // SFX eco/frequency
+    - [0x4F870, asm] // audio quality
+    - [0x4FDB0, asm] // audio engine
+    - [0x4FE60, asm] // ---UNKNOWN NO EXECUTING
+    - [0x4FF50, bin] // audio latency
+    - [0x500F8, asm] // ---UNKNOWN NO EXECUTING
+    - [0x50120, asm] // audio frequency
+    - [0x50860, asm] // title screen song flags
+    - [0x50990, asm] // ---UNKNOWN NO EXECUTING
+    - [0x50A00, asm] // title screen song setter
+    - [0x50C20, asm] // audio engine
+    - [0x50CC0, asm] // audio pointers and flags
     - [0x517A0, asm] // yay0 audio decoder
-    - [0x518A0, asm]
-    - [0x51BC0, asm]
-    - [0x51C50, asm]
-    - [0x520C0, asm]
-    - [0x52AC0, asm]
-    - [0x53150, asm]
-    - [0x535C0, asm]
-    - [0x53710, asm]
-    - [0x54310, asm]
-    - [0x558C0, asm]
-    - [0x55960, asm]
-    - [0x55BB0, asm]
-    - [0x55ED0, asm]
-    - [0x56170, asm]
-    - [0x56680, asm]
-    - [0x567B0, asm]
-    - [0x569F0, asm]
-    - [0x57190, asm]
-    - [0x57230, asm]
-    - [0x572D0, asm]
+    - [0x518A0, asm] // audio pointers and flags
+    - [0x51BC0, asm] // ---UNKNOWN NO EXECUTING
```

Comment:
For the ones that are unknown, you can just leave those empty for now. we'll add info there when we have it